packages/fxa-settings/src/pages/Signin/SigninRecoveryMethod/Phone/ConfirmRecoveryCode/en.ftl

```diff
@@ -0,0 +1,12 @@
+## ConfirmRecoveryCode page
+
+confirm-recovery-code-flow-heading = Sign in
+confirm-recovery-code-with-code-heading = Enter recovery code
```

Comment:
Could you add a comment explaining what a recovery code is? I think this is the first instance of using this term.

Reply:
Agreed, we have so many different codes now... in a comment we can say it's a one-time use SMS text message code.